commerce_coordinator/apps/ecommerce/views.py

```diff
@@ -57,3 +60,45 @@ def get(self, request):
         )
 
         return Response(format_signal_results(results))
+
+
+class OrderCreateView(APIView):
+    """Accept Ecommerce request for creating a basket/order for a user."""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+    throttle_classes = (UserRateThrottle,)
+
+    def get(self, request):  # pylint: disable=inconsistent-return-statements
+        """
+        Create orders for an authenticated user.
+
+        Args:
+            product_sku: Array. An edx.org stock keeping units (SKUs) that the user would like to purchase.
+            coupon_code: (Optional) A coupon code to initially apply to the order.
+            edx_lms_user_id: (Temporary) Initially we will be calling this API from a server. this param is to bypass
+                the edx_lms_user_id from the calling server. later on, we will remove this param and extract
+                edx_lms_user_id from request.user.lms_user_id.
+
+        Returns:
+            order_created_save:
+                error: Boolean: Represents if there was error in releasing signal
+
+        Errors:
+            400: if required params are missing or not in supported format.
+            401: if user is unauthorized.
+
+        """
+        order_created_signal_params = {
+            'product_sku': request.query_params.getlist('product_sku'),
+            # TODO: edx_lms_user_id should be taken from request.user.lms_user_id once we go live.
+            'edx_lms_user_id': request.query_params.get('edx_lms_user_id'),
+            'email': request.user.email,
```

Comment:
We may need to get the email from the query_params too, at least for this first pass.